Automationexercise/PageObjects/CheckoutPage.cs

```diff
@@ -0,0 +1,72 @@
+using AutomationExercise;
+using AutomationExercise.Models;
+using Microsoft.Playwright;
+
+namespace Automationexercise.PageObjects;
+
+public class CheckoutPage (IPage page, string baseUrl) : BasePage(page: page, baseUrl)
+{
+    //Selectors & Locators
+    private ILocator OrderComment => Page.Locator("[name='message']");
+    private ILocator PlaceOrderButton => Page.Locator(".btn[href='/payment']");
+    
+    //Methods
+    public async Task<bool> IsAddressDetailsVisible(object user) 
+    {
+        if (user == null) throw new ArgumentNullException(nameof(user));    
+        
+        var deliveryAddress = await Page.Locator("//ul[@id=\"address_delivery\"]/li").AllAsync();
+        if (deliveryAddress == null) throw new InvalidOperationException("Address details are not visible");
+        var userDetails = new List<string>
+        {
+            $"{((User)user).Title} {((User)user).FirstName} {((User)user).LastName}",
```

Comment:
so you can remove all extra symbols and instead of
$"{((User)user).Title} {((User)user).FirstName} {((User)user).LastName}",
write just
$"{user.Title} {user.FirstName} {user.LastName}",

Reply:
fixed